entity-service-impl/src/main/java/org/hypertrace/entity/service/util/DocStoreConverter.java

```diff
@@ -89,8 +93,18 @@ private static Filter transform(AttributeFilter filter) {
       f.setFieldName(filter.getName());
       f.setOp(transform(filter.getOperator()));
       if (filter.hasAttributeValue()) {
-        transform(filter.getAttributeValue(), f, isPartOfAttributeMap(filter.getName()));
+        //transform(filter.getAttributeValue(), f);
+        if (ATTRIBUTES_LABELS_FIELD_NAME.equals(f.getFieldName()) && f.getOp() == Op.EQ) {
```

Comment:
So, because we don't have type concept in ES today, we are hard-coding this for only labels attribute right?

Reply:
yes